eng/common/scripts/Verify-Links.ps1

```diff
@@ -18,13 +18,18 @@ param (
   # the substitute branch name or SHA commit
   [string] $branchReplacementName = "",
   # flag to allow checking against azure sdk link guidance. Check link guidance here: https://aka.ms/azsdk/guideline/links
-  [bool] $checkLinkGuidance = $false
+  [bool] $checkLinkGuidance = $false,
+  # UserAgent to be configured for web request. Default to PSUserAgent. 
+  [string] $userAgent = "Microsoft.PowerShell.Commands.PSUserAgent"
 )
 
 $ProgressPreference = "SilentlyContinue"; # Disable invoke-webrequest progress dialog
 # Regex of the locale keywords.
 $locale = "/en-us/"
 $emptyLinkMessage = "There is at least one empty link in the page. Please replace with absolute link. Check here for more information: https://aka.ms/azsdk/guideline/links"
+if (!$userAgent) {
```

Comment:
We should probably either default the value here or in the parameter but not both.

Reply:
It is needed for code flow. Will remove the parameter defaults.